src/Event/ReadModel/History/EventPlaceHistoryProjector.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\ReadModel\History;
+
+use Broadway\EventHandling\EventListener;
+use CultureFeed_Cdb_ParseException;
+use CultuurNet\UDB3\Cdb\EventItemFactory;
+use CultuurNet\UDB3\Event\Events\EventCopied;
+use CultuurNet\UDB3\Event\Events\EventCreated;
+use CultuurNet\UDB3\Event\Events\EventImportedFromUDB2;
+use CultuurNet\UDB3\Event\Events\EventUpdatedFromUDB2;
+use CultuurNet\UDB3\Event\Events\LocationUpdated;
+use CultuurNet\UDB3\Event\Events\MajorInfoUpdated;
+use CultuurNet\UDB3\Event\ReadModel\Relations\EventLocationHistoryRepository;
+use CultuurNet\UDB3\EventHandling\DelegateEventHandlingToSpecificMethodTrait;
+use CultuurNet\UDB3\Model\Place\PlaceIDParser;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Model\ValueObject\Web\Url;
+use CultuurNet\UDB3\ReadModel\DocumentDoesNotExist;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use Psr\Log\LoggerInterface;
+
+class EventPlaceHistoryProjector implements EventListener
+{
+    use DelegateEventHandlingToSpecificMethodTrait;
+
+    private EventLocationHistoryRepository $repository;
+    private DocumentRepository $eventRepository;
+    private LoggerInterface $logger;
+
+    public function __construct(
+        EventLocationHistoryRepository $eventLocationHistoryRepository,
```

Comment:
The naming is still inconsistent, there is a mix of `EventPlaceHistoryProjector` and `EventLocationHistoryRepository`

Reply:
I made it consistent, and then also made methods / the table name consistent.